organization/_posts/2024/2024-09-26-coordination.md

```diff
@@ -0,0 +1,167 @@
+
+We believe that input coordinated by our **activity areas** will be critical to doing this, especially from the critical areas of event generation, simulation, reconstruction and analysis; however, we should not at all forget the *people* dimension, so training and careers are vital to cover.
+
+Feedback from working groups...
+
+- Simulation have discussed their possible inputs - how to proceed with them now?
+    - Possibly discuss week before CHEP
+- Evaluation of new programming languages should be mentioned
+
+Michel and Graeme agreed to help coordinating this work but other volunteers are most welcome (required!) to help.
+
+- M&G to organise a dedicated discussion on this topic in October
+
+### LHCC
+
+
+
+### HSF Seminar Series
+
+Seminar coordinators: Benedikt, Michel, Andrea
+
+First HSF Seminar will be organised as part of JuliaHEP:
+
+- Uwe Hernandez Acosta, *Julia in high-energy physics: a paradigm shift or just another tool?* 1 October at 16h30 CEST, <https://indico.cern.ch/event/1452314/>
+
+We would like to organise further seminars on 
+
+- 30 October
+- 27 November
+- 29 January
+- 26 February (CERN local school holidays)
+
+Current topic suggestions:
+
+- Software Triggers & Reco proposed something around 4-D reconstruction
+- Event generators on GPUs for the November seminar (after CHEP)
+
+---
+
+## Steering Group News
+
+There was a Steering Group meeting on 23 September and the [minutes](https://hepsoftwarefoundation.org/organization/2024/09/23/steering.html) are available on the website.
+
+- HSF Affiliated Projects proposal was circulated:
+    - [document](https://docs.google.com/document/d/1Un1V21LdehQNwkNGeUZOl_GBQ8IdjUpDp5bi9g2YvLg/edit?usp=sharing)
+    - [guidelines on best practices and badges](https://docs.google.com/document/d/1AiLcqyLA4c1y2Iq-YZyKP7DwN8m2AJb_J44cDuGGAXI/edit)
+    - After feedback, put this onto the website and start to approach projects
+- The HSF Advisory Group [document](https://docs.google.com/document/d/1L62pleuuME6K9WLl5T4KQaImPP_Qpc8HSym-XBNTKv8/edit?usp=sharing) is finalised
+    - Will put this on the website and contact the experiments concerning constituting the AG
+
+---
+
+## Activity Updates
+
+Our working groups and activities have been merged into a single "Activities" area on the website ([PR](https://github.com/HSF/hsf.github.io/pull/1582)).
+
+"Reviews" sits a bit awkwardly here (and it's not even complete) so needs a revamp.
+
+Areas which were actually not active at all were moved to a [*archived* area](https://hepsoftwarefoundation.org/what_are_activities.html#archive) (Analysis Facilities Forum, Conditions Databases, Differentiable Computing, Frameworks, Visualisation).
+
+### Data Analysis
+
+- Next meeting on Oct 28: https://indico.cern.ch/event/1461088/, focused on Scipp (https://scipp.github.io/) and ragged data. Room also booked at CERN, email to DAWG sent yesterday (feel free to advertise to interested communities!).
```

Comment:
The links need to be enclosed in `<..>`s or they do not appear as links on the webpage